### DISCUSSION TITLE
Quantized Embeddings

### DISCUSSION THREAD

**Discussion by snadeem1362** [open-webui/open-webui#3083] [General]
**Is your feature request related to a problem? Please describe.**
Hello... we are using open-webui RAG for our company documents. we have almost 500GBs of data. Our system is generating embeddings and it is resulting in huge size of chroma db file. it is too much time consuming and require HW as well.
we are also incorporating wikipedia articles. Is there any builtin solution available so that we dont have to regenerate embeddings?
At the moment all-minilm-L6-v2 is generating 12GB of charoma DB against 1GB of wikipedia text.

**Describe the solution you'd like**
Can we add a feature to enable Quantize Embeddings. Binary quantization reduces size and speedup the retreival process.

**Comment by rahepler2** [comment]
I would also like to see this implemented. If we could use binary quantization to get the rough sources, then use those souces in the rag with Hybrid search it would allow a much faster rag experience overall.
